oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/introspection/NimbusOpaqueTokenIntrospectorTests.java

```diff
@@ -119,15 +122,15 @@
 			+ "     }";
 	// @formatter:on
 
-	private static final ResponseEntity<String> ACTIVE = response(ACTIVE_RESPONSE);
+	private static final ResponseEntity<String> ACTIVE = jsonResponse(ACTIVE_RESPONSE);
```

Comment:
This change doesn't seem necessary in order to achieve the desired improvements. In order to keep these commits focused on improving introspection's behavior, will you remove changes like these?

Reply:
Yep, totally forgot about this, as originally I had additional code to deal with something else in here too (was a couple of weeks ago now so I don't remember the full details, but I know I worked out a way around this). Will revert. Thanks for pointing this out.

Reply:
Should be reverted now.